app/components/result/ConfirmedResult.tsx

```
@@ -0,0 +1,113 @@
+  const [selectedDays, setSelectedDays] = useState<string[]>([]);
+  const [isTimePickerOpen, setIsTimePickerOpen] = useState(false);
+  const [isFirstDayPickerOpen, setIsFirstDayPickerOpen] = useState(false);
+  const [isTimeVariesByDay, setIsTimeVariesByDay] = useState(false);
+  const [bookInfo, setBookInfo] = useState("");
+
+  const toggleDay = (day: string) => {
+    setSelectedDays((prev) =>
+      prev.includes(day) ? prev.filter((d) => d !== day) : [...prev, day],
+    );
+  };
+  const handleToggleTimeVaries = (id: string) => {
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_+$/u.

(@typescript-eslint/no-unused-vars)